.gitignore

```diff
@@ -1,6 +1,7 @@
 
 packages/api/node_modules
 packages/app/node_modules
+packages/api/env/
```

Comment:
Do we need this?

Reply:
I think so. This will be needed since the `credentials.json` lives inside of env. But if that is redundant then we can leave it out.